src/ansys_sphinx_theme/announcement_layout.html

```diff
@@ -0,0 +1,16 @@
+{% if theme_switcher %} 
+<div class="container-xl">
+    <div class="row"></div>
+        <div class="col-12 col-md-10" id="header-announcement">
+        <div class="col-12 col-md-13 col-xl-7" id="header-announcement-spacer">
+            <p id="announce"></p>
+            <script>
+            let file = "announcement.txt"
```

Comment:
I just realized that it is not possible to create a link through a "txt" file in an easy way. Thus, let us revert this to an HTML file. Sorry fort the mess:

```suggestion
            let file = "announcement.html"
```

Reply:
I am not optimistic about this file path neither... What happens when this gets included in the header of a file that lives in a higher depth than the "announcement.html" file? I bet it does not find it and thus, does not include its content.